src/app/workspace/screen/tabs.tsx

```diff
@@ -17,30 +16,53 @@ import { ReactComponent as SquareIcon } from "../../assets/icons/tab/square.svg"
 import { ReactComponent as ActionsIcon } from "../../assets/icons/tab/actions.svg";
 import { ReactComponent as AddIcon } from "../../assets/icons/add.svg";
 import * as constants from "../../appconst";
+import { Reorder, AnimatePresence } from "framer-motion";
 
 import "../workspace.less";
 import "./tabs.less";
 
 dayjs.extend(localizedFormat);
 
+const TAB_WIDTH = 175;
```

Comment:
move this constant into magiclayout.ts.  i'm collecting all of the "magic" or constant layout stuff in that file so we have a list of everything in the future.  will help when we have to support "zooming" of the interface.